src/main/webapp/app/exercises/programming/manage/grading/feedback-analysis/feedback-analysis.component.ts

```
@@ -4,31 +1,168 @@
+    readonly collectionsSize = computed(() => this.content().numberOfPages * this.pageSize());
+
+    readonly faSort = faSort;
+    readonly faSortUp = faSortUp;
+    readonly faSortDown = faSortDown;
+    readonly faFilter = faFilter;
+    readonly faUpRightAndDownLeftFromCenter = faUpRightAndDownLeftFromCenter;
+    readonly SortingOrder = SortingOrder;
+    readonly MAX_FEEDBACK_DETAIL_TEXT_LENGTH = 150;
+    readonly sortIcon = computed(() => (this.sortingOrder() === SortingOrder.ASCENDING ? this.faSortUp : this.faSortDown));
+
+    hasAppliedFilters: boolean = false;
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)